pool/src/pool/submit.rs

```diff
@@ -48,6 +49,25 @@ pub fn execute_submit(
         panic_with_error!(e, PoolError::InvalidHf);
     }
 
+    // deal with potential flash lending before dealing with the other transfers.
+    // tbd: do we want to allow for multiple flash loans? There doesn't seem to be a risk in it so currently enabled.
```

Comment:
I agree.

The only real use case I can imagine for this would probably not benefit from multiple contracts being invoked, however.

It seems more likely that a user would flash borrow multiple tokens to 1 contract to then invoke. However, this is would break `moderc3156` interface.

IMO we can leave for now, but it's worth some additional thought.

cc: @markuspluna